VERSIONS.md

```diff
@@ -1,5 +1,6 @@
 | Version | iOS version | Android version | Common files version |
 | ------- | ----------- | --------------- | -------------------- |
+| 2.4.0   | 3.14.1      | 4.6.0           | 2.0.0                |
```

Comment:
I think this needs updating. I did a quick check and I believe it should be this:
```suggestion
| 2.4.1   | 3.14.0      | 4.6.1           | 2.0.0                |
```

Reply:
updated. I also updated to 2.0.1, I missed that version before